python/lsst/ts/MTDome/mock_controller.py

```diff
@@ -483,7 +483,7 @@ async def config(self, system, settings):
             # DM-26653: The structure of the settings variable has changed so
             # we need slightly more complex code here. For info about the new
             # structure, see the project docs at
-            # https://ts-dome.lsst.io/protocols.html#configuration-protocol
+            # https://ts-mtdome.lsst.io/protocols.html#configuration-protocol
```

Comment:
The ticket DM-26653 referenced above has already been marked Done. But a ticket number sounds like a "to do".

I'm not convinced you need any comment here (configuration is documented in the schema itself and in the standard place in the CSC docs). But if you really need one I'd stick to "For information about the configuration see https://..." or similar.

For future reference: when the config schema changes in a non-backwards-compatible way you should update the version number (in the title and correspondingly in the mt_config_xxx package), but I am not suggesting you do that now and it's probably not crucial until the package is released.

Reply:
Thanks. I added the ticket reference so it was clear where the modification was introduced. But I agree that that is open to interpretation so I removed the entire comment. Thanks for informing me about updating the version of the schema.